templates/release-statefulset.yaml

```diff
@@ -365,6 +371,14 @@ spec:
             {{- if .Values.extraVolumeMounts }}
             {{- include "common.tplvalues.render" (dict "value" .Values.extraVolumeMounts "context" $) | nindent 12 }}
             {{- end }}
+            {{- if .Values.logback.fileLogging }}
+            - name: logback
+              mountPath: /opt/xebialabs/xl-release-server/conf/logback.xml
+              subPath: logback.xml
+            - name: data
+              mountPath: /opt/xebialabs/xl-release-server/log
+              subPath: log
```

Comment:
The target is to use other volume as storage, not the data. Can we use extraVolumeMounts and extraVolumes to make a configuration for the logs volume?